src/django_components/template_loader.py

```diff
@@ -47,47 +50,11 @@ def get_dirs(self) -> List[Path]:
                     f"See Django documentation. Got {type(component_dir)} : {component_dir}"
                 )
                 continue
-            curr_directories: Set[Path] = set()
-
-            # For each dir in `settings.STATICFILES_DIRS`, we go over all Django apps
-            # and, for each app, check if the STATICFILES_DIRS dir is within that app dir.
-            # If so, we add the dir as a valid source.
-            # The for loop is based on Django's `get_app_template_dirs`.
-            for app_config in apps.get_app_configs():
```

Comment:
Removed the IF logic on this line (https://github.com/EmilStenstrom/django-components/pull/533/files#diff-756de4c35ec676b6ad61e551979238b310a7806c7c612add84aaec6f376c1b16L65):
```py
if hasattr(settings, "BASE_DIR"):
```

So that we treat STATICFILES_DIRS as full paths. This means that if people previously specified relative paths in STATICFILES_DIRS, they will now have to set them to full paths themselves.

Also removed the IF logic on this line https://github.com/EmilStenstrom/django-components/pull/533/files#diff-756de4c35ec676b6ad61e551979238b310a7806c7c612add84aaec6f376c1b16L71
```py
if settings.SETTINGS_MODULE:
```

Instead of relying on `SETTINGS_MODULE`, we now use only STATICFILES_DIRS and BASE_DIR. So if people used `SETTINGS_MODULE` before, they will have to set `BASE_DIR` instead.